v20.2/create-schedule-for-backup.md

```diff
@@ -0,0 +1,222 @@
+- Ensure you are monitoring your backup schedule (e.g., [Prometheus](monitor-cockroachdb-with-prometheus.html)) and alerting metrics that will confirm that your backups are completing, but also that they're not running more concurrently than you expect.
+- Ensure that your [GC window](configure-replication-zones.html#gc-ttlseconds) is long enough to accommodate your backup schedule, otherwise your incremental backups will fail. For example, if you set up your schedule to be `RECURRING '@daily'` but your GC window is less than 1 day, all your incremental backups will fail.
+
+## View and control backup schedules
+
+Once a backup schedule is successfully created, you can do the following:
+
+ Action                | SQL Statement
+-----------------------+-----------------
+View the schedule      | [`SHOW SCHEDULES`](show-schedules.html)
+Pause the schedule     | [`PAUSE SCHEDULES`](pause-schedules.html)
+Resume the schedule    | [`RESUME SCHEDULES`](resume-schedules.html)
+Drop the schedule      | [`DROP SCHEDULES`](drop-schedules.html)
+
+## View and control a backup initiated by a schedule
+
+After CockroachDB successfully initiates a scheduled backup, it registers the backup as a job. You can do the following with each individual backup job:
+
+ Action                | SQL Statement
+-----------------------+-----------------
+View the backup status | [`SHOW JOBS`](show-jobs.html)
+Pause the backup       | [`PAUSE JOB`](pause-job.html)
+Resume the backup      | [`RESUME JOB`](resume-job.html)
+Cancel the backup      | [`CANCEL JOB`](cancel-job.html)
+
+You can also visit the [**Jobs** page](admin-ui-jobs-page.html) of the Admin UI to view job details. The `BACKUP` statement will return when the backup is finished or if it encounters an error.
+
+## Examples
+
+### Create a schedule for full backups only (core)
+
+Core users can only use backup scheduling for full backups:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE SCHEDULE core_schedule_name
+  FOR BACKUP INTO 's3://test/schedule-test-core?AWS_ACCESS_KEY_ID=x&AWS_SECRET_ACCESS_KEY=x'
+    WITH revision_history RECURRING '@daily'
+    FULL BACKUP ALWAYS;
+~~~
+~~~
+     schedule_id     |        name        | status |         first_run         | schedule |                                                                                       backup_stmt
+---------------------+--------------------+--------+---------------------------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  588799238330220545 | core_schedule_name | ACTIVE | 2020-09-11 00:00:00+00:00 | @daily   | BACKUP INTO 's3://test/schedule-test-core?AWS_ACCESS_KEY_ID=x&AWS_SECRET_ACCESS_KEY=x' WITH revision_history, detached
+(1 row)
+~~~
+
+To use the other backup scheduling features, you need an [enterprise license](enterprise-licensing.html).
```

Comment:
This is slightly misleading -- they can take database full backups, for example and they can use the SCHEDULE OPTIONS. Although I don't have a suggestion for how to make it better D: